thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -815,16 +787,12 @@ public Command getCommand(
     @NonNull
     @WorkerThread
     public Trigger postNewTrigger(
```

Comment:
Non `TriggeredCommandForm` version `postNewTrigger` and `patchTrigger` were removed in thing-if iOSSDK.
If trait adaptable thing-if AndroidSDK becomes version 1.0.0, Could we remove such duplicated methods like thing-if iOSSDK done?